### DISCUSSION TITLE
Replace `gibberish.random123@domain.com` with `gibberish-random123@domain.com`

### DISCUSSION THREAD

**Discussion by ghost** [simple-login/app#1752] [Feature Requests]
I suggest replacing `gibberish.random123@domain.com` with `gibberish-random123@domain.com` or `gibberis_random123@domain.com`

I think it would be a good idea to keep the points working even after this change without making any drastic changes to the user interface.

For example:

`gibberish-random123@domain.com` = `g.i.b.b.e.r.i.s.h-random123@domain.com` but NOT the `g.i.b.b.e.r.i.s.h-r.a.n.d.o.m.1.2.3@domain.com`

Where `gibberish` is any word the user enters and `random123` is a random word or 5 letters.

What is this for?
Firstly it will protect you from being detected by some sites with a very crude e-mail address policy.
Secondly, it will allow you to keep working as points (maybe someone will be convenient)
Thirdly, it will allow to create several aliases right on the fly (it will be very convenient especially for free accounts where the quantity of aliases is limited; for example if you decide to create an alias for mailing lists you can create ONE alias, but to supplement it with points for different mailers)

As @Nypheena mentioned some websites detects dots. So I think this will solve the problem. @nguyenkims i hope you will answer?

**Comment by ShovelMail** [comment]
I think it will be really useful because some sites blocks dots as that think it is something like gmail alias

**Reply by zyxzxyx** [reply]
Instead of replace, give users options to choose their preferred separator; dot, dash or underscore.
